crates/bevy_ecs/src/query/access.rs

```diff
@@ -2,6 +2,10 @@ use crate::storage::SparseSetIndex;
 use fixedbitset::FixedBitSet;
 use std::marker::PhantomData;
 
+/// `Access` keeps track of read and write accesses to [`SparseSetIndices`](SparseSetIndex).
+///
+/// This type is useful for calculating whether two different `Access`es collide and what the
```

Comment:
```suggestion
/// This type is useful for calculating whether two different `Access`'s collide and what the
```

This is a bit of a clearer approach to pluralization here IMO.

Reply:
I disagree - I've never seen a use of apostrophe to pluralise with which I've agreed with. Ideally, we'd rework the sentence to avoid the plural here entirely, such as by using `... [Access] values collide`

This idea was taken from https://stackoverflow.com/a/41370606

(I can't 'properly' suggest this as mobile makes it sufficiently difficult to copy and paste the comment text that I can't be bothered)